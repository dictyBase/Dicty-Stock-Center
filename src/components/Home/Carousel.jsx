import React, { Component } from 'react'
import { ResponsiveImage, Figure } from 'styles'

export default class Carousel extends Component {
    displayName = 'front page carousel'
    render() {
        return (
            <Figure>
              <ResponsiveImage
                src="http://wiki.dictybase.org/dictywiki/images/c/cd/DG1100.jpg"
              />
              <figcaption>
                  The mutant pictures shown here here
                  have been provided by Bill Loomis.
                  Many mutants are available at the DSC
              </figcaption>
            </Figure>
        )
    }
}

