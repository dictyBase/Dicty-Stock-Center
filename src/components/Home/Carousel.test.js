import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Carousel from './Carousel'

it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Carousel />, div)
})

test('matching a snapshot of Carousel', () => {
    const component = renderer.create(<Carousel />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
