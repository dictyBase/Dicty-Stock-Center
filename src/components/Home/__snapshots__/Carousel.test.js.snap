// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot of Carousel 1`] = `
<figure>
  <img
    className="sc-bZQynM dnzfow"
    src="http://wiki.dictybase.org/dictywiki/images/c/cd/DG1100.jpg"
  />
  <figcaption>
    The mutant pictures shown here here have been provided by Bill Loomis. Many mutants are available at the DSC
  </figcaption>
</figure>
`;
